packages/commonwealth/client/scripts/controllers/app/webWallets/cosmos_evm_metamask_web_wallet.ts

```diff
@@ -75,7 +76,7 @@ class CosmosEvmWebWalletController implements IWebWallet<string> {
     return this._chainId;
   }
 
-  public async signCanvasMessage(account: Account, canvasMessage: CanvasData): Promise<string> {
+  public async signCanvasMessage(account: Account, canvasMessage: SessionPayload): Promise<string> {
```

Comment:
Maybe we should rename this `signSessionPayload` or `signCanvasSessionPayload`

Reply:
I'd like to do a full pass over the names when integrating Joel's new serialization helpers that just landed (maybe in this PR, maybe in a separate PR depending on the order in which different things get finished/reviewed)